src/compressed_tensors/utils/safetensors_load.py

```diff
@@ -283,11 +283,9 @@ def get_quantization_state_dict(model_path: str) -> Dict[str, Tensor]:
     weight_mappings = get_weight_mappings(model_path)
     state_dict = {}
     for weight_name, safe_path in weight_mappings.items():
-        if not is_quantization_param(weight_name):
+        if is_quantization_param(weight_name):
+            state_dict[weight_name] = safe_path
```

Comment:
Can you rename this function and its type hint? Calling a map from weight_paths to file_paths is not what a state_dict is